component/prometheus/exporter/windows/config.go

```diff
@@ -0,0 +1,190 @@
+package windows
+
+import (
+	windows_integration "github.com/grafana/agent/pkg/integrations/windows_exporter"
+)
+
+// DefaultArguments holds non-zero default options for Arguments when it is
+// unmarshaled from YAML.
+//
+// Some defaults are populated from init functions in the github.com/grafana/agent/pkg/integrations/node_exporter package.
+var DefaultArguments = Arguments{
```

Comment:
This is missing some defaults; see `pkg/integrations/windows_exporter/config_windows.go` for init functions which modify the default arguments in the older integration.

For specific collectors to work correctly (like SMTP), we need an equivalent of that init function here, or we need to manually populate the defaults from windows_exporter rather than trying to automatically infer them from Kingpin flags on startup.